README.md

```diff
@@ -1,2 +1,6 @@
 # Hamilton
 Program controlling game 
+# Description
+program operating on a database to control and to administrate during the match taking place in a real world
+# Objective
+Using mechanics of databases create the most simply and common available procedure for even beginning users
```

Comment:
`simply` - adverb
and
`common` - adjective
Something went wrong.
And I would change the word `available` to `possible`, but it's my opinion